e2e/scenario_test.go

```diff
@@ -1137,9 +1137,8 @@ func Test_ubuntu2204WasmAirGap(t *testing.T) {
 				nbc.OutboundType = datamodel.OutboundTypeBlock
 				nbc.ContainerService.Properties.SecurityProfile = &datamodel.SecurityProfile{
 					PrivateEgress: &datamodel.PrivateEgress{
-						Enabled: true,
-						// TODO(xinhl): create one private acr instead of mcr.microsoft.com
-						ContainerRegistryServer: "mcr.microsoft.com",
+						Enabled:                 true,
+						ContainerRegistryServer: "aksvhdtestcr.azurecr.io/aks",
```

Comment:
@xinhl / @AlisonB319 I though that there wouldnt be any remote/cache name that we would be receiving ?

Reply:
From your question on 7/13 it seems to still be open ended - https://teams.microsoft.com/l/message/19:5c77d666c90e4cfb98b66a226f74d957@thread.v2/1721762320450?context=%7B%22contextType%22%3A%22chat%22%7D

I've pinged Xhine, so he should see this, this evening.